src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -65,7 +66,7 @@
 
 	boolean index() default true;
 
-	DateFormat format() default DateFormat.none;
+	DateFormat[] format() default { DateFormat.date_optional_time, DateFormat.epoch_millis };
 
 	String pattern() default "";
```

Comment:
@sothawo not sure if we want to support multiple custom format patterns as well.

Reply:
why not?

Reply:
Ok, I also added support for multiple custom formats.